### DISCUSSION TITLE
No textarea support ?

### DISCUSSION THREAD

**Discussion by EthanRDM-dev** [KingSora/OverlayScrollbars#504] [Q&A]
Hi ! I was redirected to this plugin from the plugin SimpleBar because it doesn't support the textarea scrollbar and its creator said this one have supports the textarea. Maybe I'm not doing this correctly, right now this is my code to set the OverlayScrollbar on my textarea :

```
OverlayScrollbars(document.querySelector("#myTextarea"), {});
```

I'm using Mozilla Firefox 109.0.1, jQuery CDN 3.6.3 and the overlayscrollbars.browser.es5.min.js distribution.

Thanks in advance, and sorry if I just missed something !

**Comment by KingSora** [comment]
Good day @EthanRDM-dev 

`v1` of OverlayScrollbars supports the `textarea` element. In `v2` this feature is missing but its on the roadmap.

**Reply by Malstrime** [reply]
Looking forward for textarea plugin <3
You're doing a great job

**Reply by artashes-sanoyan** [reply]
Just to confirm, support for `textarea` is not there yet right ? I've checked release notes coming after `v2` did not find `textarea` being mentioned.

**Reply by KingSora** [reply]
@artashes-sanoyan thats correct

**Reply by punkpeye** [reply]
Still no textarea support?

**Reply by 1951FDG** [reply]
Also looking forward to this feature!
